css/style.css

```diff
@@ -0,0 +1,40 @@
+body{
+    background-image: url('../images/transform.jpg');
```

Comment:
Please use a better background image and make it transparent. A good background image should be visually appealing and should also convey the message of your content